content/docs/user_guide.md

```diff
@@ -0,0 +1,765 @@
++++
+title = "User Guide"
+description = "In-depth instructions for using Kubeflow"
+weight = 10
+toc = true
+bref= "Detailed description of Kubeflow usage"
++++
+
+**Table of Contents**
+
+- [Using Kubeflow](#using-kubeflow)
+  - [Requirements](#requirements)
+  - [Deploy Kubeflow](#deploy-kubeflow)
+    - [Usage Reporting](#usage-reporting)
+    - [Bringing up a Jupyter Notebook](#bringing-up-a-jupyter-notebook)
+    - [Serve a model using TensorFlow Serving](#serve-a-model-using-tensorflow-serving)
+    - [Serve a model using Seldon](#serve-a-model-using-seldon)
+    - [Submitting a TensorFlow training job](#submitting-a-tensorflow-training-job)
+      - [Run the TfCnn example](#run-the-tfcnn-example)
+    - [Submitting a PyTorch training job](#submitting-a-pytorch-training-job)
+      - [Run the Pytorch MNIST example](#run-the-pytorch-mnist-example)
+  - [Advanced Customization](#advanced-customization)
+  - [Troubleshooting](#troubleshooting)
+    - [TensorFlow and AVX](#tensorflow-and-avx)
+    - [Minikube](#minikube)
+    - [RBAC clusters](#rbac-clusters)
+    - [OpenShift](#openshift)
+    - [Docker for Mac](#docker-for-mac)
+    - [403 API rate limit exceeded error](#403-api-rate-limit-exceeded-error)
+  - [Why Kubeflow Uses Ksonnet](#why-kubeflow-uses-ksonnet)
+
```

Comment:
Please remove lines 9-31. The Table of Contents will get auto generated.

Reply:
Updated the PR